src/modules/claim/services/claim.service.ts

```diff
@@ -269,6 +269,32 @@ export class ClaimService {
     return qb.getMany();
   }
 
+  /**
+   * Get claims able to be revoked by user with matching DID
+   * @param filters additional filters
+   * @param currentUser revoker's DID
+   */
+  async getByRevoker({
+    filters: { namespace } = {},
+    currentUser,
+  }: {
+    filters?: QueryFilters;
+    currentUser?: string;
```

Comment:
Does it make sense for `currentUser` to be optional? Seems like this method doesn't really work without it.